<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmDeletion_Window</class>
 <widget class="QDialog" name="ConfirmDeletion_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <widget class="QPushButton" name="btn_yes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>是</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_no">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>否</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>是否删除该邮件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
